### DISCUSSION TITLE
Calificacion de Actividad S3P2

### DISCUSSION THREAD

**Discussion by gonaiot** [ISPC-TST-EM2022/Grupo7#35] [Announcements]
Califiacion Grupal: 7

@chulkx : 8
@NahuelOtero : 7
@ruizgb : 7
@Darman82 : 6

Observacion: Chicos trabajaron muy bien en forma colectiva y ademas son muy uniformes en la participacion. Continuen asi y aprovechen estas instancias para mejorar su produccion de codigo.

**Comment by chulkx** [comment]
Muchas gracias por la devolución profe.
